opus/application/static_media/js/opus.js

```diff
@@ -646,13 +659,17 @@ var opus = {
         $(document).on("keydown click", function(e) {
             if ((e.which || e.keyCode) == 27) {
                 // ESC key - close modals and help panel
-                $(".op-confirm-modal").modal('hide');
+                // But don't close "#op-browser-version-msg" and "#op-browser-size-msg"
+                if (!$(".op-confirm-modal").is("#op-browser-version-msg") &&
+                    !$(".op-confirm-modal").is("#op-browser-size-msg")) {
+                    $(".op-confirm-modal").modal("hide");
+                }
                 opus.hideHelpPanel();
             }
         });
 
         // Handle the Submit or Cancel buttons for the various confirm modals we can pop up
-        $(".op-confirm-modal").on("click", ".btn", function() {
+        $(".op-confirm-modal").on("click", ".btn", function(event) {
```

Comment:
while i think it's good form to add the parameter event, i'm not sure i see where you are using it...  I also think it might be good to standardize on 'e' vs 'event' as a name, we use both.